app/views/notes/_comment.html.erb

```diff
@@ -18,6 +18,17 @@
     </div>
 
     <div class="navbar-text navbar-right pull-right" style="padding-right:10px;">
+      <% if comment.author && comment.author.user && comment.author.user.role %>
+        <% streak = comment.author.user.streak
+           if streak[0] > 7 %>
+          <a href="/wiki/streaks"><i class="fa fa-fire" style="font-size:16px; color:#FF0000;" data-toggle="tooltip" data-placement="top" title="You are on Fire! <%= streak[1][0]%> notes, <%= streak[1][1]%> wiki edits, <%= streak[1][2]%> comments in last <%= streak[0]%> days"></i></a>
+        <% end %>
+        <% if comment.author.user.role == 'admin' %>
```

Comment:
Let's just comment the role icon stuff out for now -- pending further design discussion on #215. Thanks, and sorry it wasn't quite at a design state to be implemented yet! Let's not let it hold up the streaks function.